egs/swbd/s5c/local/chain/run_tdnn.sh

```diff
@@ -1 +1 @@
-tuning/run_tdnn_7e.sh
\ No newline at end of file
+tuning/run_tdnn_7g.sh
```

Comment:
@danpovey @tomkocse @freewym Do you actually want to make this the preferred swbd recipe ?